ui/ducks/send/send.js

```diff
@@ -109,6 +109,7 @@ import {
   isBurnAddress,
   isPossibleAddress,
   isValidHexAddress,
+  isPossibleAddress,
```

Comment:
This change probably needs to be removed for lint to pass

Reply:
Sure done.